engine/src/engine.rs

```diff
@@ -2762,7 +2762,7 @@ mod tests {
         let mut handler = Noop;
         let actual_result = refund_on_error(io, &env, expected_state, &args, &mut handler).unwrap();
         let expected_result =
-            SubmitResult::new(TransactionStatus::Succeed(Vec::new()), 21344, Vec::new());
+            SubmitResult::new(TransactionStatus::Succeed(Vec::new()), 21860, Vec::new());
```

Comment:
Why did this EVM gas amount change?

Reply:
It's related to EIP-7623 as the gas floor cost was increased